### PR TITLE
travis integration

### PR DESCRIPTION
**Pull request by v-stepanov**
As there are tests I think it makes sense to integrate CI

### CODE REVIEW

**Comment by adyach** [comment]
👍

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: python
+
+python:
+  - "3.5"
+
+install:
+  - pip install bubuku
```
Comment (antban) [inline .travis.yml]: pip3 install bubuku

Comment (antban) [inline .travis.yml]: And as far as I understand - install section is needed to install dependencies, not project by itself.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+script:
+  - py.test tests
```
Comment (antban) [inline .travis.yml]: Can you please explain what is in there?
I thought that for testing one need to do something like this: 
```
python3 setup.py test
```

**Comment by antban** [comment]
Rejecting because of PR #14

**Pull request closed without merging**